walnut/cmd/convert.go

```diff
@@ -56,6 +57,9 @@ specify which modules should be used when generating task templates. Control-OCC
 		for _, dumpFile := range args {
 			// Strip .json from end of filename
 			nameOfDump := dumpFile[:len(dumpFile)-5]
+			vars := map[string]string {
```

Comment:
Looks good, just please call this one `defaults`, because the contents of the map end up in `defaults` in the TT.